.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -1,52 +1,50 @@
+  cy.fixture('customers').then((customer) => {
+    cy.get('#input-firstname').type(
+      `${customer.newCustomer.firstName}.${random}`
     );
-    cy.get('#input-payment-lastname').type(
-      `${guest.newGuest.lastName}.${random}`
+    cy.get('#input-lastname').type(
+      `${customer.newCustomer.lastName}.${random}`
     );
-    cy.get('#input-payment-email').type(`${guest.newGuest.email}`);
-    cy.get('#input-payment-telephone').type(guest.newGuest.phone);
-    cy.get('#input-payment-address-1').type(`${guest.newGuest.address}`);
-    cy.get('#input-payment-city').type(guest.newGuest.city);
-    cy.get('#input-payment-postcode').type(guest.newGuest.postCode);
-    cy.get('#input-payment-zone').select(guest.newGuest.zone);
+    cy.get('#input-email').type(`${customer.newCustomer.email}.${random}`);
+    cy.get('#input-password').type(`${customer.newCustomer.password}`);
+    cy.get('#input-shipping-address-1').type(`${customer.newCustomer.address}`);
+    cy.get('#input-shipping-city').type(customer.newCustomer.city);
+    cy.get('#input-shipping-postcode').type(customer.newCustomer.postCode);
+    cy.get('#input-shipping-zone').select(customer.newCustomer.zone);
   });
-  cy.get('#button-guest').click();
-  cy.get('#button-shipping-method').click();
-  cy.get('[name="agree"]').check();
-  cy.get('#button-payment-method').click();
-  cy.get('#button-confirm').click();
-  cy.contains('order has been placed');
+  cy.get('#agree').click();
+  cy.get('#button-register').click();
+  cy.contains('Your account has been created');
   cy.login();
-  cy.contains('Sales').click();
-  cy.contains('Orders').click();
-  cy.fixture('guests').then((guest) => {
-    cy.contains(`${guest.newGuest.firstName}.${random}`);
+  cy.get('#menu-customer').click();
+  cy.get('#menu-customer > ul').within(() => {
+    cy.contains('Customers').click({ force: true });
+  });
+  cy.fixture('customers').then((customer) => {
+  cy.contains(`${customer.newCustomer.email}`);
   });
 });
 
 it('allows an admin to add a product to the catalog', () => {
   cy.login();
   cy.contains('Catalog').click();
   cy.contains('Product').click();
-  cy.get('[data-original-title="Add New"]').click();
+  cy.get('[title="Add New"]').click();
   cy.fixture('products').then((product) => {
-    cy.get('#input-name1').type(`${product.newProduct.name}.${random}`);
-    cy.get('#input-meta-title1').type(product.newProduct.metatagTitle);
+    cy.get('#input-name-1').type(`${product.newProduct.name}.${random}`);
+    cy.get('#input-meta-title-1').type(`${product.newProduct.metatagTitle}.${random}`);
     cy.contains('Data').click();
-    cy.get('#input-model').type(product.newProduct.model);
-    cy.get('[data-original-title="Save"]').click();
+    cy.get('#input-model').type(`${product.newProduct.model}.${random}`);
+    cy.get('#input-keyword-0-1').type(`${product.newProduct.model}.${random}`, { force: true });
+    cy.get('button.btn.btn-primary i.fa-solid.fa-floppy-disk').click();
```

Comment:
Same as the `Add New` button, this selector should be replaceable by `[title="Add New"]/[aria-label="Add New"]`. Probably the first one as it has the same "on hover" logic.